treeherder/etl/perf.py

```diff
@@ -67,7 +67,8 @@ def _get_signature_hash(signature_properties):
 
 
 def load_perf_artifacts(project_name, reference_data, job_data, datum):
-    perf_datum = json.loads(datum['blob'])
+    blob = json.loads(datum['blob'])
+    perf_datum = blob['performance_data']
```

Comment:
Yeah, we do assume this. If this happened in the field, we'd return a 500 error.

This reminds me that autophone will need some small updates to support the fact that the performance artifact is now wrapped in a dictionary (its key being performance_data). I can provide a patch.